README.md

```diff
@@ -1,233 +1,249 @@
+| CONFIG_ADMIN_ENDPOINT            | URL that provides metadata about the B2C tenant's OpenID Connect configuration, such as the issuer URL, token signing keys, and supported scopes. This is for the admin journey.        |No|
+| CONFIG_ENDPOINT                  | Same as above but for media journey.                                                                                                                                                    |No|
+| MEDIA_VERIFICATION_CONFIG_ENDPOINT| Same as above but for verification of media accounts.                                                                                                                                   |No|
+| OIDC                             | Boolean referring to whether the service is running secure mode or not.                                                                                                                 |No|
+| SESSION_SECRET                   | Unique identifier or value that's used to identify a user's session - can really be any string if you're running locally.                                                               |Yes|
+| FRONTEND_URL                     | This is the host that the service uses to identify what it's running on. Defaults to staging, but you want it to be `https://localhost:8080` if you're running locally (in secure mode) |No|
+| REDIS_HOST                       | Hostname of utilised Redis instance                                                                                                                                                     |No|
+| REDIS_PORT                       | Port that utilised Redis instance is running on                                                                                                                                         |No|
+| REDIS_LOCAL                      | Boolean to determine if Redis runs locally or not                                                                                                                                       |No|
+| TENANT_ID                        | Directory unique ID assigned to our Azure AD tenant. Represents the organisation that owns and manages the Azure AD instance.                                                           |No|
+| DATA_MANAGEMENT_AZ_API           | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-data-management service                                |No|
+| ACCOUNT_MANAGEMENT_AZ_API        | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-account-management service                             |No|
+| SUBSCRIPTION_MANAGEMENT_AZ_API   | Used as part of the `scope` parameter when requesting a token from Azure. Used for service-to-service communication with the pip-subscription-management service                        |No|
```

Comment:
Think it is missing CHANNEL_MANAGEMENT_AZ_API

Reply:
Added